tests/bridge/setup.rs

```diff
@@ -123,13 +125,19 @@ pub async fn setup_test() -> SetupConfig {
         &HashMap::from([
             (
                 CommitmentMessageId::Superblock,
-                WinternitzPublicKey::from(&commitment_secrets[&CommitmentMessageId::Superblock]),
+                WinternitzPublicKeyVariant::Standard(WinternitzPublicKey::from(
```

Comment:
Here as well we have a double `WinternitzPublicKey` conversion in all hashmap elements.

Reply:
DONE